scripts/tests_coverage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Butterfly build root
+BUTTERFLY_BUILD_ROOT=$1
+
+client_obj_dir=$BUTTERFLY_BUILD_ROOT/api/client/CMakeFiles/api_client.dir/
+server_obj_dir=$BUTTERFLY_BUILD_ROOT/api/server/CMakeFiles/api_server.dir/
+
+# Test Butterfly build root
+if [ ! -d $client_obj_dir -o ! -d $server_obj_dir ]; then
+    echo "Please set butterfly's build root as parameter"
+    exit 1
+fi
+
+# Test coverage data files
+ret=$(find $BUTTERFLY_BUILD_ROOT/api/ -name '*.gcda' | wc -l)
+if [ $ret -eq 0 ]; then
+    echo " Coverage files (*.gcda) not found in $client_obj_dir"
+    echo " Make sure you did that in your build directory:"
+    echo " - cmake .. -DDEBUG=ON"
+    echo " - make"
+    echo " run some tests before running coverage test."
+    exit 1
+fi
+
+rm -fr $BUTTERFLY_BUILD_ROOT/client_coverage
```

Comment:
I generally use -rfv when I'm using an rm, like this if something goes wrong in this test, the user might notice it before it's too late